Conf/Package/package.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# This script packages up Santa and its configs.
+# The output is "${SCRATCH}/app.pkg".
+#
+# If BUILD_DEV_DISTRIBUTION_PKG is set, "santa-dev.pkg" will also be produced.
+# The dev package is helpful when testing installer behavior for dev builds.
+#
+# All of the following environment variables are required.
+
+# RELEASE_ROOT is a required environment variable that points to the root
+# of an extracted release tarball produced with the :release and :release_driver
+# rules in Santa's main BUILD file.
+[[ -n "${RELEASE_ROOT}" ]] || die "RELEASE_ROOT unset"
+
+[[ -n "${SCRATCH}" ]] || die "SCRATCH unset"
```

Comment:
nit: would be nice to comment this as required above with `RELEASE_ROOT`

Reply:
Good call. When writing the comment, I realized `SCRATCH` really isn't required. Instead, what we were using it for should have its own variable. `PKG_OUT_DIR` - added a comment.